diesel/src/query_builder/functions.rs

```diff
@@ -124,3 +124,10 @@ pub fn select<T>(expression: T) -> Select<SelectStatement<()>, T> where
 {
     SelectStatement::simple(()).select(expression)
 }
+
+/// Creates an insert statement. Will insert default values in the table. This
+/// function is not exported by default. As with other commands, the resulting
+/// query can return the inserted rows if you choose.
```

Comment:
The problem with moving the test here is that I don't have access to schema_dsl which means 3 SQL queries to create the table with default values.